contracts/interchain-token-service/src/event.rs

```diff
@@ -56,6 +76,42 @@ impl Event for TrustedChainRemovedEvent {
     }
 }
 
+impl Event for InterchainTokenDeployedEvent {
+    fn topics(&self, env: &Env) -> impl Topics + Debug {
+        (
+            Symbol::new(env, "interchain_token_deployed"),
+            self.token_id.to_val(),
+            self.token_address.to_val(),
+            self.minter.clone(),
+            self.name.to_val(),
+            self.symbol.to_val(),
+            self.decimals,
+        )
+    }
+
+    fn data(&self, env: &Env) -> impl IntoVal<Env, Val> + Debug {
+        Vec::<Val>::new(env)
+    }
+}
+
+impl Event for InterchainTokenDeploymentStartedEvent {
+    fn topics(&self, env: &Env) -> impl Topics + Debug {
+        (
+            String::from_str(env, "interchain_token_deployment_started"),
```

Comment:
```suggestion
            Symbol::new(env, "interchain_token_deployment_started"),
```

Reply:
Let's use a smaller symbol then. The first topic should always be a symbol for consistency. String looks different in XDR

Maybe `token_deployment_started`

Reply:
got it, updated to use `token_deployment_started`